test/test_distributions.py

```diff
@@ -7,7 +7,7 @@
 import torch
 from common import TestCase, run_tests
 from torch.autograd import Variable, gradcheck
-from torch.distributions import (Bernoulli, Beta, Categorical, Dirichlet,
+from torch.distributions import (Bernoulli, Beta, Categorical, Chi2, Dirichlet,
                                  Exponential, Gamma, Laplace, Normal)
 from torch.distributions.uniform import Uniform
```

Comment:
Would you mind cleaning up the `Uniform` import, moving it into `from torch.distributions import ...`?